app/controllers/messages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class MessagesController < ApplicationController
+
+  def index
+    @received_messages = current_customer.received_messages
```

Comment:
It looks like me and Danny both needed the Customer associations so we pushed this file up in another commit